CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add events correlation engine plugin ([#6854](https://github.com/opensearch-project/OpenSearch/issues/6854))
 - Add connectToNodeAsExtension in TransportService ([#6866](https://github.com/opensearch-project/OpenSearch/pull/6866))
 - Adds ExtensionsManager.lookupExtensionSettingsById ([#7466](https://github.com/opensearch-project/OpenSearch/pull/7466))
+- Add RenameResponseProcessor for Search Pipelines ([#7377](https://github.com/opensearch-project/OpenSearch/pull/7377))
```

Comment:
Could you please move it to `## [Unreleased 2.x]` section? (I believe we will backport it to `2.x`) Thank you.

Reply:
Added in [a5ce249](https://github.com/opensearch-project/OpenSearch/pull/7377/commits/a5ce24972f239d4c03d91dd11d31c3749d1c7c5a)